modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,296 @@
+  { start: '*', cmd: 'InsertUnorderedList', trigger: 'space' },
+  { start: '-', cmd: 'InsertUnorderedList', trigger: 'space' },
+  { start: '>', cmd: 'mceBlockQuote', trigger: 'space' },
+  { start: '---', cmd: 'InsertHorizontalRule', trigger: 'space' },
+]
+----
+
+For information on the **text_patterns**, see xref:content-behavior-options.adoc#text_patterns[Text Patterns].
+
+=== Removed
+
+==== The autocompleter `ch` configuration property has been removed. Use the `trigger` property instead.
+// #TINY-8929
+
+Previously, the `ch` configuration property was used to specify the character that would trigger the autocompleter.
+
+This property was deprecated in link:https://www.tiny.cloud/docs/tinymce/6/6.2-release-notes/#deprecated[{productname} 6.2] and has been removed in {productname} 7.0. Instead, use the `trigger` property to specify the `string` that will trigger the autocompleter.
+
+If `+editor.ui.registry.addAutocompleter(name, options)+` was used in your configuration, updating your configuration from `ch: '<string>',` to `trigger: '<string>',` is required.
+
+NOTE: The new `trigger` option can handle multiple character strings as the trigger.
+
+For more information, visit the updated xref:autocompleter.adoc[Autocompleter] documentation.
 
 [[options]]
 == Options
 
-...
+=== Changed
+
+[[highlight-on-focus]]
+==== `highlight_on_focus`
+
+Previously, the default value for the editor configuration option `highlight_on_focus` was set to `false` by default. In {productname} 7.0 this option is now by default set to `true`.
+
+As a result, the focus outline for the editor will by default be displayed when the focus is on the editor.
+
+Customers who have created their own custom skins, or have implemented their own outline and do not intend to use the default will need to add `highlight_on_focus: false` to their {productname} configuration.
+
+[NOTE]
+This change does not impact editors using inline mode.
+
+=== Removed
+
+[[force-hex-color]]
+==== `force_hex_color`
+
+Previously in {productname} 6, all colors in the content HTML were set to use `rgb` values by default. As the common practice is using `hex` values this change has been reverted.
+
+.Example
+[source, html]
+----
+<!-- example using RGB values -->
+<p>Hello <span style="color: rgb(255, 0, 0);">red</span> color</p>
+
+<!-- example using HEX values -->
+<p>Hello <span style="color: #ff0000">red</span> color</p>
+----
+
+In {productname} 7.0, all colors (excluding `rgba` values) are converted to use `hex` values and the `forced_hex_color` option has been removed.
 
 [[plugins]]
 == Plugins
 
-...
+=== Changed
+
+==== Changed the `media_url_resolver` option to use promises.
+// #TINY-9154
+
+In {productname} 6, the `media_url_resolver` option used a callback rather than a promise, which lead to unforeseen consequences.
+
+To rectify this, all instances of `media_url_resolver` were refactored to utilize promises in {productname} 7.
```

Comment:
```suggestion
In {productname} 6 and earlier, the `media_url_resolver` option provided `resolve` and `reject` callbacks, rather than a Promise. In certain circumstances this implementation resulted in unexpected behavior.

In {productname} 7, the `media_url_resolver` option now requires a Promise to be returned.
```